internal/init/init.go

```diff
@@ -170,8 +144,8 @@ func writeConfig(config string, b []byte) error {
 }
 
 type Servicer interface {
-	API(*cobra.Command) *packngo.Client
-	ListOptions(defaultIncludes, defaultExcludes []string) *packngo.ListOptions
+	MetalAPI(*cobra.Command) *metal.APIClient
```

Comment:
Unless we _have_ to change this to `MetalAPI` to avoid errors, I'd say keep it as `API`
```suggestion
	API(*cobra.Command) *metal.APIClient
```

Reply:
As you know we are keeping packngo and metal-go in. we plan in seperate PR to rename MetalAPI -> API at the time we deprecated the packngo completely

Reply:
Ah, right, this needs to match the function name in `root.go`